specs/binding-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/schema/kafka.schema.patch.json

```diff
@@ -78,6 +78,166 @@
                                         {
                                             "type": "string",
                                             "enum": [ "none", "json_patch" ]
+                                        },
+                                        "key":
+                                        {
+                                            "type": "object",
+                                            "additionalProperties": false,
+                                            "properties":
+                                            {
+                                                "type":
+                                                {
+                                                    "title": "Type",
+                                                    "type": "string",
+                                                    "enum":
+                                                    [
+                                                        "byte_buffer",
+                                                        "bytes",
+                                                        "double",
+                                                        "float",
+                                                        "integer",
+                                                        "avro",
+                                                        "json",
+                                                        "json_decoder",
```

Comment:
What is the difference in validation between `json_decoder` and `json`?

Reply:
json_decoder is required in case, we want to transform avro or proto event to json. But since we are planning to have transformation piece separately, it can be considered then.